### PR TITLE
Create edit mission modal

### PR DESCRIPTION
**Pull request by IamGideonIdoko**
- [ ]  Created a modal for editing mission  #383

### CODE REVIEW

**Comment by KelvinAkpobome** [comment]
Please resolve conflict

**Comment by KelvinAkpobome** [comment]
You don't seem to have any issue linked to this PR, why so please?

**Comment by KelvinAkpobome** [comment]
Please resolve conflicts

File: client/src/components/modal/EditMission.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+const SaveBtn = styled.button`
+  border: none;
+  background-color: rgba(0, 184, 124, 1);
+  color: white;
+  width: 120px;
+  height: 30px;
+  border-radius: 6px;
+  cursor: pointer;
+  float: right;
+`;
```
Comment (Asylcreek) [inline client/src/components/modal/EditMission.jsx:110]: Are you sure you shouldn't move these styles into a separate file?

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: client/src/components/modal/EditMission.jsx (hunk 1 above)
Comment (IamGideonIdoko) [reply client/src/components/modal/EditMission.jsx]: Ahh. I've moved the styles to a separate file.

**Review by Asylcreek — CHANGES_REQUESTED** [review]
Please rename "styledEditMission.js" to "EditMission.styled.js".

**Pull request merged**